### PR TITLE
Rebecca joyner

### PR DESCRIPTION
**Pull request by rebeccajoyner**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,16 +4,95 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Vibur" rel="stylesheet">
+    
+  <link href="https://fonts.googleapis.com/css?family=McLaren" rel="stylesheet">
+
+    </style>
+    </head>
+    
+    <body>
+    
+    <div id="header"></div>
```
Comment (Zealll) [inline index.html:27]: Why do you have empty `divs` here?

File: less/index.css [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+h1 {
+  padding: 5px;
+}
+h2 {
+  padding: 5px;
+  font-size: 1.5rem;
+}
+.left {
+  height: 100%;
+  width: 45px;
```
Comment (Zealll) [inline less/index.css:163]: try to use rems instead of px

File: less/index.css [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  float: left;
+  position: fixed;
+}
+@media (max-width: 500px) {
+  .left {
+    display: none;
+  }
+}
+.right {
+  height: 1000px;
```
Comment (Zealll) [inline less/index.css:174]: I would switch this to `min-height: 1000px`

File: less/index.css [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+  background-color: #778899;
+  float: right;
+}
+@media (max-width: 500px) {
+  .right {
+    display: none;
+  }
+}
+.stuff {
+  display: inline-block;
```
Comment (Zealll) [inline less/index.css:185]: Let's leave `inlin-block` in the past, and use `flex` :P

File: less/index.css [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+  text-decoration-style: double;
+  text-decoration-color: #DB7093;
+}
+p,
+li {
+  font-family: 'McLaren', cursive;
+  padding: 1px;
+}
+.head {
+  font-size: 20px;
```
Comment (Zealll) [inline less/index.css:208]: `rems` here as well

File: less/index.css [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+}
+.nav {
+  border: 2px dotted #DB7093;
+  font-size: 1rem;
+  text-align: center;
+  padding: 5px 5%;
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 10px;
```
Comment (Zealll) [inline less/index.css:217]: Would recommend switching this either to `rems` or to `%`

File: less/index.css [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+    border: 2px dotted #778899;
+    padding: 0.5rem;
+  }
+}
+#name {
+  font-family: 'Vibur', cursive;
+  float: right;
+  margin-top: 10px;
+  margin-right: 4%;
+  font-size: 1.9rem;
```
Comment (Zealll) [inline less/index.css:230]: BEAUTIFUL!!!! :D

**Review by Zealll — APPROVED** [review]
You have been doing awesome Rebecca!!! Keep it up!!! Tho sometimes you are too quiet, I think you could engage with your group mates even more :)

**Comment by rebeccajoyner** [comment]
thank you! I'll work on that. sometimes it takes me a little while to warm up to a new group. Next week I will make it a priority tho!

**Pull request merged**